### DISCUSSION TITLE
人大金仓连接问题

### DISCUSSION THREAD

**Discussion by 113774172** [dotnetcore/FreeSql#1044] [Q&A]
由于业务需要，需要适配达梦以及人大金仓库，目前达梦适配完成了，没有问题，但适配人大金仓过程中一直报以下错误：
【主库】状态不可用，等待后台检查程序恢复方可使用。版本字符串部分太短或太长。
 感觉报的这个错没啥参考性，无法排查问题，看有人遇到过这个问题没

**Comment by hd2y** [comment]
先尝试使用 `ADO.NET` 直连，检查是否是连接字符串的问题，这个大概率不是 FreeSql 抛出来的错，只是被 FreeSql 拦截到了。

```
Driver={KingbaseES 8.2 ODBC Driver ANSI};Server=127.0.0.1;Port=54321;UID=USER2;PWD=123456789;database=TEST
```
